sfts/s3_to_sfts.py

```diff
@@ -161,19 +225,24 @@ def is_processed():
 try:
     xfer_jar = f"{xfer_path}/xfer.jar"
     jna_jar = f"{xfer_path}/jna.jar"
-    print(("trying to call subprocess:\nxfer.jar: "
+    logger.info(("trying to call subprocess:\nxfer.jar: "
            f"{xfer_jar}\njna.jar : {jna_jar}"))
     subprocess.check_call(
         ["java", "-classpath", f"{xfer_jar}:{jna_jar}",
          "xfer",
          f"-user:{sfts_user}",
          f"-password:{sfts_pass}",
          f"-s:{sfts_conf}",
-         "filetransfer.gov.bc.ca"])
+         "filetransfer.gov.bc.ca"],
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL)
```

Comment:
From https://docs.python.org/3/library/subprocess.html#subprocess.DEVNULL

> Special value that can be used as the stdin, stdout or stderr argument to Popen and indicates that the special file os.devnull will be used.

If the output is to the null device, I think we can't ever really review what the output was. Correct me if I'm wrong.

We want to keep the log of the output of the subprocess call somewhere for review.